src/panels/config/zha/ha-config-zha.ts

```diff
@@ -34,18 +51,68 @@ export class HaConfigZha extends LitElement {
               icon="hass:arrow-left"
               @click="${this._onBackTapped}"
             ></paper-icon-button>
+            <div main-title="">Zigbee Home Automation</div>
```

Comment:
I don't think you have to set main-title...but not 100% sure
```suggestion
            <div main-title>Zigbee Home Automation</div>
```